float32[6220800] image
---
libuavf_2024/Detection2D[] detections